CODEOWNERS

```diff
@@ -461,7 +461,6 @@
 /include/display/                         @vanwinkeljan
 /include/dt-bindings/clock/kinetis_mcg.h  @henrikbrixandersen
 /include/dt-bindings/clock/kinetis_scg.h  @henrikbrixandersen
-/include/dt-bindings/dma/stm32_dma.h      @cybertale
```

Comment:
This change should be moved to another appropriate commit

Reply:
moved to commit a6a6897b79e1640192697e45197a834be64d99ff